### PR TITLE
Add TeX to unicode module

### PR DESCRIPTION
**Pull request by Tyilo**

### CODE REVIEW

File: tex_to_unicode.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    if not success:
+        item.text = stripped
+        item.subtext = 'Type some TeX math'
+        success = False
+
+    if success:
+        item.addAction(ClipAction('Copy result to clipboard', result))
+
+    return [item]
```
Comment (ManuelSchneid3r) [inline tex_to_unicode.py]: here too

**Comment by Tyilo** [comment]
It now returns the item directly. Ready to be merged?

**Comment by Tyilo** [comment]
Ping @ManuelSchneid3r

**Comment by ManuelSchneid3r** [comment]
Is it possible to do reverse too? I mean like µ -> \mu? Well but the kbd does not have a lot symbols anyway.

**Pull request merged**